modules/core/src/commonMain/kotlin/fr/acinq/lightning/channel/Commitments.kt

```diff
@@ -190,7 +191,7 @@ sealed class LocalFundingStatus {
         override val fee: Satoshi = sharedTx.tx.fees
     }
 
-    data class ConfirmedFundingTx(override val signedTx: Transaction, override val fee: Satoshi, val localSigs: TxSignatures) : LocalFundingStatus() {
+    data class ConfirmedFundingTx(override val signedTx: Transaction, override val fee: Satoshi, val localSigs: TxSignatures, val shortChannelId: ShortChannelId) : LocalFundingStatus() {
```

Comment:
Since we are not currently using the `scid`, I would probably have left it nullable and not do the update trick until we really need it. But your way makes the model cleaner.

Reply:
I would have left it nullable _until_ we use it (since it may never happen?)